tensorflow_io/arrow/python/ops/arrow_dataset_ops.py

```diff
@@ -88,7 +87,7 @@ def arrow_schema_to_tensor_types(schema):
   return tensor_types, tensor_shapes
 
 
-class ArrowBaseDataset(data.Dataset):
+class ArrowBaseDataset(dataset_ops.DatasetV2):
```

Comment:
Is this ok to use as a base for 1.15.0?

Reply:
That should be OK I think. As long as dataset could be passed to tf.keras (with 1.15 and 2.0) we should be fine. Both DatasetV1 and DatasetV2 works with tf.keras. 👍